### PR TITLE
DK starting zone and dragonflight route update

### LINKED ISSUE #121: [ISSUE] - Scarlet Enclave route and Dragonflight route update
## A23-ScarletEnclave and H23-ScarletEnclave Route
[Google Sheets route ](https://docs.google.com/spreadsheets/d/1qppw9dl-etV2-NtKtzfjie4GGGUKU0QuUh5abWwOT-M/)

Please also add to Transport.db - i suggest right before the Dragonflight section around line 832
```
-- Eastern Kingdoms
    if (APR.ActiveMap == "A23-ScarletEnclave" and (APRt_Zone == 37 or APRt_Zone == 84 or APRt_Zone == 124)) then
        APRt_Zone = 23
    elseif (APR.ActiveMap == "H23-ScarletEnclave" and (APRt_Zone == 1 or APRt_Zone == 85 or APRt_Zone == 124)) then
        APRt_Zone = 23
    end
```

Note: you will need to modify your script, if you haven't already, to account for the PickUpDB / QpartDB / DoneDB steps
Steps 104/105/106

 ## Dragonflight 
 Also, `DF05-2024-AzureSpan` needs to be re-exported. Found a typo in a questID and fixed it just now.

### PR DESCRIPTION
**Pull request by Neogeekmo**
CF localization✅ 

close #121

### CODE REVIEW

**Review by Rycia — APPROVED** [review]

File: APR-Core/Core.lua [hunk 1]
```diff
@@ -77,98 +77,79 @@ function APR.AutoPathOnBeta(routeChoice) -- For the Speed run and First characte
...
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"DEV - StoryMode Only (Not Enough XP)")
-	elseif (routeChoice >= 1 and APR.Level >= 60 and APR.Faction == "Alliance") then
-		autoPatchStartInsertRoute()
+	autoPatchStartInsertRoute()
+	if(APR.Level < 10 and (ZeMap == 1409 or ZeMap == 1726 or ZeMap == 1727)) then
+		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"01-10 Exile's Reach")
+	end
+	if(APR.Level < 30) then
+		if(APR.Class[3] == 6) then
+			tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"08-30 Dark Knight Start")
```
Comment (Pahonix) [inline APR-Core/Core.lua:86]: Should be "Death Knight"

File: APR-Core/zones/Alliance.lua [hunk 2]
```diff
@@ -12,6 +12,7 @@ if (APR.Faction == "Alliance") then
 		 ["01-30 Demon Hunter Start"] = 672,
 		 ["01-30 Dun Morogh (Full)"] = 27,
 		 ["01-30 Elwynn Forest (Full)"] = 37,
+		 ["08-30 Dark Knight Start"] = 23,
```
Comment (Pahonix) [inline APR-Core/zones/Alliance.lua:15]: Should be "Death Knight"

File: APR-Core/zones/Alliance.lua [hunk 3]
```diff
@@ -108,6 +109,8 @@ if (APR.Faction == "Alliance") then
 	}
 	if (APR.Class[3] == 12) then
 		 APR.QuestStepListListing["MISC 1"]["672-Mardum"] = "01-30 Demon Hunter Start"
+	elseif (APR.Class[3] == 6) then
+			APR.QuestStepListListing["MISC 1"]["A23-ScarletEnclave"] = "08-30 Dark Knight Start"
```
Comment (Pahonix) [inline APR-Core/zones/Alliance.lua:113]: Should be "Death Knight"

File: APR-Core/zones/Horde.lua [hunk 4]
```diff
@@ -13,6 +13,7 @@ if (APR.Faction == "Horde") then
 		 ["01-30 Demon Hunter Start"] = 672,
 		 ["01-10 Exile's Reach"] = 1409,
 		 ["01-30 Durotar (Full)"] = 1,
+		 ["08-30 Dark Knight Start"] = 23,
```
Comment (Pahonix) [inline APR-Core/zones/Horde.lua:16]: Should be "Death Knight"

File: APR-Core/zones/Horde.lua [hunk 5]
```diff
@@ -84,6 +85,8 @@ if (APR.Faction == "Horde") then
 	}
 	if (APR.Class[3] == 12) then
 		 APR.QuestStepListListing["MISC 1"]["672-Mardum"] = "01-30 Demon Hunter Start"
+	elseif (APR.Class[3] == 6) then
+			APR.QuestStepListListing["MISC 1"]["H23-ScarletEnclave"] = "08-30 Dark Knight Start"
```
Comment (Pahonix) [inline APR-Core/zones/Horde.lua:89]: Should be "Death Knight"

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua [hunk 6]
```diff
@@ -1,3 +1,1640 @@
...
+        {	-- Step 104
+             ["PickUp"] = {
+                  12742,
+             },
+             ["TT"] = {
+                  ["y"] = 1375.8,
+                  ["x"] = -5713.9,
+             },
+             ["PickUpDB"] = {
+                  12743,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua:1191]: Insert "12742," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua [hunk 7]
```diff
@@ -1,3 +1,1640 @@
...
+                  ["y"] = 1328.7,
+                  ["x"] = -5753.2,
+             },
+             ["Trigger"] = {
+                  ["y"] = 1328.7,
+                  ["x"] = -5753.2,
+             },
+             ["Range"] = 6.65,
+             ["QpartDB"] = {
+                  12743,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua:1214]: Insert "12742," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua [hunk 8]
```diff
@@ -1,3 +1,1640 @@
...
+        {	-- Step 106
+             ["Done"] = {
+                  12742,
+             },
+             ["TT"] = {
+                  ["y"] = 1370.4,
+                  ["x"] = -5720.5,
+             },
+             ["DoneDB"] = {
+                  12743,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua:1230]: Insert "12742," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua [hunk 9]
```diff
@@ -1,3 +1,1640 @@
...
+        {	-- Step 104
+             ["PickUp"] = {
+                  12739,
+             },
+             ["TT"] = {
+                  ["y"] = 1375.8,
+                  ["x"] = -5713.9,
+             },
+             ["PickUpDB"] = {
+                  12747,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua:1191]: Insert "12739," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua [hunk 10]
```diff
@@ -1,3 +1,1640 @@
...
+                  ["y"] = 1328.7,
+                  ["x"] = -5753.2,
+             },
+             ["Trigger"] = {
+                  ["y"] = 1328.7,
+                  ["x"] = -5753.2,
+             },
+             ["Range"] = 6.65,
+             ["QpartDB"] = {
+                  12747,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua:1214]: Insert "12739," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

File: APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua [hunk 11]
```diff
@@ -1,3 +1,1640 @@
...
+        {	-- Step 106
+             ["Done"] = {
+                  12739,
+             },
+             ["TT"] = {
+                  ["y"] = 1370.4,
+                  ["x"] = -5720.5,
+             },
+             ["DoneDB"] = {
+                  12747,
```
Comment (Pahonix) [inline APR-WrathOfTheLichKing/WrathOfTheLichKing_Horde.lua:1230]: Insert "12739," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.

**Review by Pahonix — CHANGES_REQUESTED** [review]
Added a few comments for changes.

**Review by NightofStarrs — CHANGES_REQUESTED** [review]
@Neogeekmo, can you fix the 'Dark Knight' to 'Death Knight', I know it is how it is written in French. However, for consistency speaking it should be all the same/English verbiage. 

Additionally, Paho made some comments about correcting a few spots that the google sheet didn't have, or the export didn't grab.

**Review by NightofStarrs — APPROVED** [review]

**Comment by NightofStarrs** [comment]
I approved it, but I still had Merging blocked. Prefer to wait for Paho to confirm everything is good first.

**Review by Pahonix — APPROVED** [review]

**Pull request merged**